src/components/itemcontextmenu.js

```diff
@@ -346,23 +346,28 @@ define(["apphost", "globalize", "connectionManager", "itemHelper", "appRouter",
                     break;
                 case "copy-stream":
                     var downloadHref = apiClient.getItemDownloadUrl(itemId);
-                    var textArea = document.createElement("textarea");
-                    textArea.value = downloadHref;
-                    document.body.appendChild(textArea);
-                    textArea.focus();
-                    textArea.select();
-                    if (document.execCommand("copy")) {
+                    navigator.clipboard.writeText(downloadHref).then(function () {
                         require(["toast"], function (toast) {
                             toast(globalize.translate("CopyStreamURLSuccess"));
                         });
-                    } else {
-                        console.error("Failed to copy to clipboard");
-                        require(["toast"], function (toast) {
-                            toast(globalize.translate("CopyStreamURLError"));
-                        });
-                    }
-
-                    document.body.removeChild(textArea);
+                    }, function () {
+                        var textArea = document.createElement("textarea");
+                        textArea.value = downloadHref;
+                        document.body.appendChild(textArea);
+                        textArea.focus();
+                        textArea.select();
+                        if (document.execCommand("copy")) {
+                            require(["toast"], function (toast) {
+                                toast(globalize.translate("CopyStreamURLSuccess"));
+                            });
+                        } else {
+                            prompt(globalize.translate("CopyStreamURL"), downloadHref);
+                            require(["toast"], function (toast) {
+                                toast(globalize.translate("CopyStreamURLSuccess"));
+                            });
```

Comment:
Not sure we need a toast if a prompt is already used here.

Reply:
You may be right, shall I remove it?

Reply:
Go for it!

Reply:
Done!